tests/test_event_detection.py

```diff
@@ -1,4 +1,120 @@
+'''Code Tests'''
+
 # Authors: Ryan Thorpe <ryvthorpe@gmail.com>
+# Authors: Darcy Diesburg <darcy.diesburg@gmail.com>
+
+import os.path as op
+
+import numpy as np
+from scipy.io import loadmat
+
+import SpectralEvents.spectralevents as se
+
 
 def test_event_comparison():
+    '''Test if the output from MATLAB and Python event detection scripts align
+
+    Parameters
+    ----------
+    method : float
+    The event detection method (1, 2, or 3) used to detect spectral events.
+
+    Notes
+    -----
+    Currently only supports testing method 1 and 3, with method 2 to be added
+    to the Python event detection functions in the future. Python output is
+    tested against the output of MATLAB scripts from Shin et al., 2017.
+    Detection of the same number of beta events within each trial is required,
+    and tolerable margins of deviation for the other parameters.
+    '''
+
+    # Define tolerable deviance for spectral event characteristics
+    dev_max_time = 5  # +-5ms allowed for timing latency diff
```

Comment:
This is super nitpicky, but what do you think of renaming this variable to `dev_event_time`? Slightly more intuitive but feel free to disagree.